dialogflow-cx-webhook-nodejs/configure-session-parameters-enable-transition.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 Google LLC
```

Comment:
Multiline comments?

Reply:
What about them?

Reply:
I don't know if they are required, as it seems most snippets use the single-line style? A couple examples:
* https://github.com/googleapis/nodejs-vision/blob/main/samples/detect.js
* https://github.com/googleapis/nodejs-pubsub/blob/main/samples/createAvroSchema.js

Reply:
Sorry that this wasen't clear. I meant that you might consider using multiline comments for readability, instead of single-line.